kml/edit-kml-ground-overlay/src/main/java/com/esri/samples/edit_kml_ground_overlay/EditKMLGroundOverlaySample.java

```diff
@@ -48,7 +48,9 @@
 
 public class EditKMLGroundOverlaySample extends Application {
 
+  private KmlGroundOverlay kmlGroundOverlay;
   private SceneView sceneView;
+  private Slider slider;
 
   @Override
   public void start(Stage stage) {
```

Comment:
for the comment on line 64, we should clarify that the scene is an application cene to avoid confusion with the comment on line 71. Suggested: 
// set title, size and add applicationn scene to stage

Reply:
Thanks, I sometimes mix this up myself. Good spot!